packages/opentelemetry-api/src/metrics/NoopMeter.ts

```diff
@@ -90,10 +89,9 @@ export class NoopMeter implements Meter {
    * @param callback the batch observer callback
    */
   createBatchObserver(
-    _name: string,
     _callback: (batchObserverResult: BatchObserverResult) => void
-  ): BatchObserver {
-    return NOOP_BATCH_OBSERVER_METRIC;
+  ): void {
```

Comment:
the return type is void, but you are still returning the `BatchObserver` in final implementation in metrics